scripts/maintenance/migrate_project/src/models.py

```diff
@@ -0,0 +1,86 @@
+import json
+from pathlib import Path
+from typing import Optional
+from uuid import UUID
+
+from pydantic import BaseModel, Field
+from sqlalchemy import desc
+
+
+class DBConfig(BaseModel):
+    address: str
+    user: str
+    password: str
+    database: str
+
+
+class S3Config(BaseModel):
+    endpoint: Optional[str] = None
+    access_key: str
+    secret_key: str
+    bucket: str
+
+
+class SourceConfig(BaseModel):
+    db: DBConfig
```

Comment:
Why not using settings_library versions of DBConfig and S3Config instead???

Reply:
See answer below.

Reply:
interdependent? YOu mean independent?